### ISSUE TITLE
Sort by Organization

### ISSUE THREAD

**Issue by sbs01** [owncloud/contacts#649]
It would by great to sort contacts by organization.

**Comment by Julian1998** [comment]
@sbs01 Yeah I agree! You are welcome to try it on your own! If you need any help let me know
